### PR TITLE
add r-rgeos

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Comment by bgruening** [comment]
ping @daler @jdblischak

File: recipes/r-rgeos/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
```
Comment (jdblischak) [inline recipes/r-rgeos/meta.yaml]: Did you consider adding `geos` as a dependency? The [CRAN entry](https://cran.r-project.org/package=rgeos) has the following field:

> SystemRequirements: GEOS (>= 3.2.0); for building from source: GEOS from http://trac.osgeo.org/geos/; GEOS OSX frameworks built by William Kyngesburye at http://www.kyngchaos.com/ may be used for source installs on OSX.

File: recipes/r-rgeos/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    - r-sp >=1.1_0
+    - libgcc  # [not win]
+
+test:
+  commands:
+    - $R -e "library('rgeos')"  # [not win]
+    - "\"%R%\" -e \"library('rgeos')\""  # [win]
+
+about:
+  home: https://r-forge.r-project.org/projects/rgeos/ http://trac.osgeo.org/geos/
```
Comment (jdblischak) [inline recipes/r-rgeos/meta.yaml]: The CRAN entry has two URLs. I've never encountered this before. Presumably conda will still build the package fine, but Anaconda Cloud may not be sure how to display this.

**Comment by jdblischak** [comment]
For archival purposes, you may want to correct the title of the PR to `r-rgeos`.

File: recipes/r-rgeos/meta.yaml (hunk 1 above)
Comment (bgruening) [reply recipes/r-rgeos/meta.yaml]: Done!

**Comment by bgruening** [comment]
@isuruf can we skip win here?

**Pull request merged**